test/cli/run.spec.ts

```diff
@@ -188,6 +188,11 @@ async function exec(opts: Options = {}) {
   if (outputString === null) {
     outputString = out.read();
   }
+
+  // It's hard to figure these out sometimes.  Give ourselves a little help.
+  console.log("RECEIVED STDOUT:", outputString);
```

Comment:
I would be ok with leaving these in if they were wrapped by an environment variable check or something, but I don't want to see them in a normal test run.

Reply:
This is only for debugging problematic test, I'll remove it